Doc/library/persistence.rst

```diff
@@ -21,3 +21,4 @@ The list of modules described in this chapter is:
    marshal.rst
    dbm.rst
    sqlite3.rst
+   audit_events.rst
```

Comment:
I added it here in persistence page since Sphinx kept complaining with a warning that audit_events.rst should be present under a toctree like below. This can be moved to a more appropriate section.

```
Warning, treated as error:
/Users/karthikeyansingaravelan/stuff/python/cpython/Doc/library/audit_events.rst:document isn't included in any toctree
```

Reply:
👍 I'll move it under `library/python.rst`

Reply:
I actually moved it to `library/debug.rst`